.github/ISSUE_TEMPLATE/Standard-Bug-Issue-Template.md

```diff
@@ -4,67 +4,67 @@ about: Use this template for all bugs
 labels: Bug
 ---
 
-## Story
-
+# Story
 **As a < Agent, Admin, or Patient> of <STDcheck, Healthlabs, UTITreatment and/or Manage >**, I need < an expression of the correct behavior needed from the system >.
 
 ## Steps to Recreate  
-
 < A numbered list of the **exact** steps taken in the system that led to the Actual Behavior. What we want here is not how to see the bad result but rather how to **create** the bad result. Screen shots or videos can help. If there are necessary pre-conditions to running the recreation steps, these should be detailed separately. >
 
 ## Expected Behavior  
-
 < Description of what one would expect to see/have happen if the system was operating properly. Screenshots or videos can help. >
 
 ## Actual Behavior
-
 < Description of what one saw/had happened with the system operating at this moment. Screenshots or videos can help. >
 
 ## Business Value
-
 < What is the gain from the fix or the cost in not fixing it (customer dissatisfaction, loss of time, money, etc. >
 
 ## Notes
-< bulleted list of special notes or suggestions to the development team on any information pertinent to the bug. > 
+< special notes or suggestions to the development team on any information pertinent to the bug. > 
+
+# Acceptance Criteria
```

Comment:
Because this is a level 1 heading, "Subject Domain Experts" and "Proposed Solution" are both sub-sections of "Acceptance Criteria". Did you intend that?

Reply:
No, will correct

Reply:
Also caught that the Notes section for Feature was out of order with Acceptance Criteria and moved it.